modules/admin/src/resources/scss/components/_mainnav.scss

```diff
@@ -208,6 +208,8 @@
     min-width: 24px;
     width: 100%;
     max-width: 25px;
+    
+    color: map-get($grays, 400);
```

Comment:
in the same class?

Reply:
its not the same class '.mainnav-link' and '.mainnav-image-icon'